src/battle_animation.cpp

```diff
@@ -39,7 +39,7 @@ BattleAnimation::BattleAnimation(int x, int y, const RPG::Animation* animation)
 		graphic = Cache::Battle2(name);
 	}
 	else {
-		Output::Warning("Couldn't find animation: %s", name.c_str());
+		Output::Debug("Called battle animation without graphic file");
```

Comment:
I mean when name of animation is e.g. "Monster" and there is no "monster.png" in "Battle" and "Battle2" this silences that warning now. Instead of reporting "Monster" not found.
When the animation is empty "" then silencing is okay I guess.